spec/controllers/sitemap_controller_spec.rb

```diff
@@ -1,4 +1,12 @@
 RSpec.describe SitemapController, type: :controller do
+  describe 'GET robot' do
+    it 'renders robot text file' do
+      get :robots
+
+      expect(response).to be_success
```

Comment:
I would change this expectation to render template one. See the description of the spec and the expectation of it.